src/backend/aspen/api/main.py

```diff
@@ -142,15 +121,22 @@ def get_app() -> FastAPI:
         exception_handler,
     )
 
+    # Auspice endpoints don't all require authentication, they can do their own login checks.
+    _app.include_router(auspice.router, prefix="/v2/auspice")
+    _app.include_router(auspice.router, prefix="/v2/orgs/{org_id}/auspice")
     # Which routes are "ready" to accept org prefixes?
     org_routers = {
-        "auspice": auspice.router,
         "sequences": sequences.router,
         "phylo_trees": phylo_trees.router,
         "phylo_runs": phylo_runs.router,
         "samples": samples.router,
     }
     for suffix, router in org_routers.items():
+        _app.include_router(
+            router,
+            prefix="/v2/" + suffix,
+            dependencies=[Depends(require_group_membership)],
```

Comment:
Make sure any access to these endpoints is limited to members of the group in question.